app/boards/shields/romac/romac.overlay

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020 Pete Johansen, Richard Jones
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/matrix-transform.h>
+
+/ {
+	chosen {
+		zmk,kscan = &kscan0;
+		zmk,matrix_transform = &default_transform;
+	};
+
+	default_transform: keymap_transform_0 {
```

Comment:
I'm going to merge this as is, bit this transform matrix actually duplicates the default behavior if no transform matrix is specified, so could be removed. Fine to leave in though.